pkg/pillar/vault/handler_ext4.go

```diff
@@ -189,15 +190,25 @@ func (h *Ext4Handler) getRemovePolicyParams(policyID string) []string {
 	return []string{"metadata", "destroy", "--policy=" + mountPoint + ":" + policyID, "--quiet", "--force"}
 }
 
-func (h *Ext4Handler) getProtectorIDByName(vaultPath string) ([][]string, error) {
-	stdOut, _, err := execCmd(fsCryptPath, h.getStatusParams(mountPoint)...)
+func (h *Ext4Handler) getProtectorIDByName(vaultPath string) ([]string, error) {
+	ctx, err := actions.NewContextFromMountpoint(vaultPath, nil)
```

Comment:
maybe it makes sense for ctx and the vaultPath to be members of the handler struct?